### PR TITLE
Accommodating mongo urls for different environments

### PR DESCRIPTION
**Pull request by frosario**

### CODE REVIEW

**Comment by frosario** [comment]
@nathanathan When you get a chance, mind looking this PR over?

File: 0collections/geonames_proxy.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 this.grits = this.grits || {};
 
 Meteor.startup(function() {
+    if (process.env.MONGO_URL) {
+        var mongo_url = process.env.MONGO_URL + '/geonames';
```
Comment (nathanathan) [inline 0collections/geonames_proxy.js]: @frosario the MONGO_URL environment variable has a database path already, so appending one like this probably won't work.

**Comment by frosario** [comment]
@nathanathan Mind taking another look? I removed the db name like you suggested, and tested on grits-dev by manually checking out this branch.

File: .scripts/autocompleteCollections.js [hunk 2]
```diff
@@ -3,9 +3,12 @@
 // that have been applied to articles in the girder database.
 // The disease classifications are used by the autocomplete on the search page.
 
-//Leaving the localhost reference to make life easier for devs working locally
-//var mongodb = new Mongo('localhost:27017');
-var mongodb = new Mongo('172.30.2.123:27017');
+if (process.env.MONGO_URL) {
+  var mongodb = new Mongo(process.env.MONGO_URL);
```
Comment (nathanathan) [inline .scripts/autocompleteCollections.js]: I don't know if the mongo library will accept the database path present in MONGO_URL.

**Comment by nathanathan** [comment]
👍

**Pull request merged**